packages/twenty-docs/src/pages/graphql/core.tsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+import GraphQlPlayground from '../../components/graphql-playground';
+
+const CoreGraphql = () => {
+  return <GraphQlPlayground path={'graphql'} subdocName={'core'} />;
```

Comment:
It can be a little confusing to have the ability to set `path={'metadatata'}  subdocName={'core'}`, maybe we can do the correspondance in the GraphQLPlayground?